specification/ai/Face/routes.session.tsp

```diff
@@ -0,0 +1,152 @@
+using Foundations;
+
+namespace Face;
+
+alias SessionCommonDescription = "A session is best for client device scenarios where developers want to authorize a client device to perform only a liveness detection without granting full access to their resource. Created sessions have a limited life span and only authorize clients to perform the desired action before access is expired.";
+alias LivenessSessionWithVerifyDescription = """
+${SessionCommonDescription}
+
+Permissions includes...
+>
+*
+  * Ability to call /detectLivenessWithVerify/singleModal for up to 3 retries.
+  * A token lifetime of 10 minutes.
+
+> [!NOTE]
+>
+> *
+>   * Client access can be revoked by deleting the session using the Delete Liveness With Verify Session operation.
+>   * To retrieve a result, use the Get Liveness With Verify Session.
+>   * To audit the individual requests that a client has made to your resource, use the List Liveness With Verify Session Audit Entries.
+""";
+alias SessionCreationSuccess = "A successful call create a session for a client device and provide an authorization token for use by the client application for a limited purpose and time.";
+alias DeleteSessionSummary = "Delete all session related information for matching the specified session id.";
+alias DeleteSessionDescription = """
+> [!NOTE]
+> Deleting a session deactivates the Session Auth Token by blocking future API calls made with that Auth Token. While this can be used to remove any access for that token, those requests will still count towards overall resource rate limits. It's best to leverage TokenTTL to limit length of tokens in the case that it is misused.
+""";
+alias ListSessionDescription = """
+List sessions from the last sessionId greater than the 'start'.
+
+The result should be ordered by sessionId in ascending order.
+""";
+alias ListSessionAuditEntriesDescription = "Gets session requests and response body for the session.";
+
+@get
+@action("audit")
+@actionSeparator("/")
+op FaceLivenessSessionListAuditEntries<TResource extends TypeSpec.Reflection.Model> is Foundations.ResourceOperation<
+  TResource,
+  ListRequestOptions,
+  Body<LivenessSessionAuditEntry[]>,
+  ServiceTraits,
+  FaceErrorResponse
+>;
+
+interface LivenessSessionOperations {
+  @doc(
+    """
+    ${SessionCommonDescription}
+
+    Permissions includes...
+    >
+    *
+      * Ability to call /detectLiveness/singleModal for up to 3 retries.
+      * A token lifetime of 10 minutes.
+
+    > [!NOTE]
+    > Client access can be revoked by deleting the session using the Delete Liveness Session operation. To retrieve a result, use the Get Liveness Session. To audit the individual requests that a client has made to your resource, use the List Liveness Session Audit Entries.
+    """
+  )
+  @summary("Create a new detect liveness session.")
+  @returnsDoc(SessionCreationSuccess)
+  createLivenessSession is FaceResourceCreateWithServiceProvidedName<
+    LivenessSession,
+    CreateLivenessSessionParameters,
```

Comment:
Not sure we could rename this model name `CreateLivenessSessionParameters`.

 This is conflicted with RLC's naming rule `{Operations}Parameters`. We would resolve this conflict in emitter side. To non-block you it would be great if you could rename it.